### PR TITLE
how to pub xml files

### LINKED ISSUE #13147: Add a note about how to make sure settings files are published
This note was in the Generic Host doc, but it fits better in this one:

To move settings files to the output directory for publishing, specify the settings files as [MSBuild project items](/visualstudio/msbuild/common-msbuild-project-items) in the project file. The sample app moves its JSON app settings files and *hostsettings.json* with the following `<Content>` item:

```xml
<ItemGroup>
  <Content Include="**\*.json" Exclude="bin\**\*;obj\**\*" 
    CopyToOutputDirectory="PreserveNewest" />
</ItemGroup>
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 62fb594f-04bf-c4d7-c94b-6cd6d8b12d75
* Version Independent ID: 64e973ac-838d-4459-06b0-4e943bff936e
* Content: [Host and deploy ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/?view=aspnetcore-2.1)
* Content Source: [aspnetcore/host-and-deploy/index.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/host-and-deploy/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-hostdeploy**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #13147

### CODE REVIEW

File: aspnetcore/host-and-deploy/index.md [hunk 1]
```diff
@@ -23,6 +23,17 @@ In general, to deploy an ASP.NET Core app to a hosting environment:
 
 The [dotnet publish](/dotnet/core/tools/dotnet-publish) command compiles app code and copies the files required to run the app into a *publish* folder. When deploying from Visual Studio, the `dotnet publish` step occurs automatically before the files are copied to the deployment destination.
 
+## Publish settings files
+
+`*.json` files are published by default. To publish other settings files, specify them in the `<ItemGroup> <Content Include= ... />` element in the project file. The following markup publishes XML files:
```
Comment (guardrex) [inline aspnetcore/host-and-deploy/index.md:28]:
```suggestion
`*.json` files are published by default. To publish other settings files, specify them in the `<ItemGroup> <Content Include= ... />` element in the project file. The following example publishes XML files:
```

**Review by guardrex — APPROVED** [review]
Recommend "example" over "markup" because an MSBuild file doesn't markup anything (outside of the elements).

**Comment by guardrex** [comment]
btw ... See if this should be cross-linked for it ...


https://docs.microsoft.com/en-us/visualstudio/msbuild/common-msbuild-project-items?view=vs-2019#content

**Comment by Rick-Anderson** [comment]
@guardrex  much appreciated, added your link

**Pull request merged**